python/cudf_kafka/cudf_kafka/_lib/kafka.pyx

```diff
@@ -26,5 +26,5 @@ cdef class KafkaDatasource(Datasource):
                                         batch_timeout,
                                         delimiter)
 
-    cdef datasource* get_datasource(self):
+    cdef datasource* get_datasource(self) except *:
```

Comment:
If we know we're not throwing a Python exception here do we still need `except *`? I.E. to protect against `RecursionError`?

Reply:
Connected offline and we can remove the `except *` from this entirely as `unique_ptr.get()` is `noexcept` in C++ land.